build.rs

```diff
@@ -8,7 +8,7 @@ use vergen::{self, ConstantsFlags};
 
 fn main() {
     let amethyst_home =
-        Path::new(&home_dir().expect("Failed to find home directory")).join(".amethyst");
+        Path::new(&config_dir().expect("Failed to find home directory")).join("amethyst");
```

Comment:
```suggestion
        Path::new(&config_dir().expect("Failed to find config directory")).join("amethyst");
```

Or even better, have it continue the build normally instead of panicking if the `.config` dir doesn't exist.